src/components/PathwaysList/NewPathwayModal.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { FC, useCallback, useRef, memo, FormEvent } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPlus, faTimes } from '@fortawesome/free-solid-svg-icons';
+import {
+  Button,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  TextField,
+  IconButton
+} from '@material-ui/core';
+import { v4 as uuidv4 } from 'uuid';
+
+import { usePathwayContext } from '../PathwayProvider';
```

Comment:
I'm assuming we're using the same style guide for the builder. This should be an absolute import